packages/blocks/azure/src/lib/azure-cli.ts

```diff
@@ -37,7 +37,18 @@ export async function runCommand(
 }
 
 async function login(credentials: any, envVars: any) {
-  const loginCommand = `login --service-principal --username ${credentials.clientId} --password ${credentials.clientSecret} --tenant ${credentials.tenantId}`;
-
-  return await runCliCommand(loginCommand, 'az', envVars);
+  try {
+    const loginCommand = `login --service-principal --username ${credentials.clientId} --password ${credentials.clientSecret} --tenant ${credentials.tenantId}`;
+
+    return await runCliCommand(loginCommand, 'az', envVars);
+  } catch (error) {
+    let message = 'Error while login into azure: ';
+    if (String(error).includes('login --service-principal')) {
```

Comment:
you don't need the IF here,you already know it's the login command

Reply:
Hmm yea i thought about that, but i think it can still have a different error, like it could fail for a different reason inside run command